utils/build/docker/nodejs/express4-typescript.Dockerfile

```diff
@@ -30,6 +30,7 @@ CMD ./app.sh
 COPY utils/build/docker/nodejs/install_ddtrace.sh binaries* /binaries/
 
 RUN npm install
+RUN npm install "express@^4.17.2" "apollo-server-express@^3.13.0" "express-mongo-sanitize@^2.2.0"
```

Comment:
`express4-typescript` application has these dependencies already declared in its [package.json](https://github.com/DataDog/system-tests/blob/main/utils/build/docker/nodejs/express4-typescript/package.json#L19-L22). Why do we need to explicitly install them when building the image?

Reply:
Good catch, should not do this for express4-typescript

Reply:
Done